python/hsfs/feature_view.py

```diff
@@ -0,0 +1,608 @@
+                for the `host` parameter in the [`hsfs.connection()`](project.md#connection) method.
+                If set to False, the online feature store storage connector is used
+                which relies on the private IP.
+        # Returns
+            `List[list]` List of lists of feature values related to provided primary keys, ordered according to
+            positions of this features in training dataset query.
+        """
+        if self._vector_server is None:
+            self.init_serving(batch=True, external=external)
+        return self._vector_server.get_feature_vectors(self, entry, external)
+
+    def preview_feature_vector(self, external: Optional[bool] = False):
+        """Returns a sample of assembled serving vector from online feature store.
+
+        # Arguments
+            external: boolean, optional. If set to True, the connection to the
+                online feature store is established using the same host as
+                for the `host` parameter in the [`hsfs.connection()`](project.md#connection) method.
+                If set to False, the online feature store storage connector is used
+                which relies on the private IP.
+        # Returns
+            `list` List of feature values, ordered according to positions of this
+            features in training dataset query.
+        """
+        if self._vector_server is None:
+            self.init_serving()
+        return self._vector_server.get_preview_vectors(self, external, 1)
+
+    def preview_feature_vectors(self, n: int, external: Optional[bool] = False):
+        """Returns n samples of assembled serving vectors in batches from online feature store.
+
+        # Arguments
+            n: int. Number of feature vectors to return.
+            external: boolean, optional. If set to True, the connection to the
+                online feature store is established using the same host as
+                for the `host` parameter in the [`hsfs.connection()`](project.md#connection) method.
+                If set to False, the online feature store storage connector is used
+                which relies on the private IP.
+        # Returns
+            `List[list]` List of lists of feature values , ordered according to
+            positions of this features in training dataset query.
+        """
+        if self._vector_server is None:
+            self.init_serving()
+        return self._vector_server.get_preview_vectors(self, external, n)
+
+    def get_batch_data(self, start_time=None, end_time=None, read_options=None):
+        """
+        start_time: timestamp in second or wallclock_time: Datetime string. The String should be formatted in one of the
+                following formats `%Y%m%d`, `%Y%m%d%H`, `%Y%m%d%H%M`, `%Y%m%d%H%M%S`, or `%Y%m%d%H%M%S%f`.
+        end_time: timestamp in second or wallclock_time: Datetime string. The String should be formatted in one of the
```

Comment:
From the comment it is not clear that the date formats can contain a "/", "-", " ", or ":" as separatort between Y m and d. 

It might be more user-friendly and reliable to got with a standard implementation for time formats rather than the check_timestamp_format_from_date_string function. I think it's okay like this for now, but we should keep it as a TODO.